src/SprykerEco/Zed/Unzer/Business/Credentials/UnzerCredentialsValidator/UnzerCredentialsStoreRelationsValidator.php

```diff
@@ -48,6 +48,10 @@ public function __construct(UnzerReaderInterface $unzerReader)
     public function validate(UnzerCredentialsTransfer $unzerCredentialsTransfer): UnzerCredentialsResponseTransfer
     {
         $unzerCredentialsResponseTransfer = (new UnzerCredentialsResponseTransfer())->setIsSuccessful(true);
+        if (in_array((int)$unzerCredentialsTransfer->getTypeOrFail(), UnzerConstants::UNZER_CHILD_CONFIG_TYPES, true)) {
```

Comment:
required transfer fields should be listed at API specs

Reply:
added to specs